### PR TITLE
added code of conduct

### PR DESCRIPTION
**Pull request by OrionStar25**
Also, changed title, added logo in the title bar.

### CODE REVIEW

**Comment by Anupam-dagar** [comment]
LGTM
Can you provide a live link.

**Comment by OrionStar25** [comment]
https://hackcodeofconduct.org/530-hack_in_the_north_2018 - link for code of conduct

**Comment by OrionStar25** [comment]
https://orionstar25.github.io/hackinthenorth.github.io/main/ - here.

**Comment by gauthamzz** [comment]
@OrionStar25 Now about section is not fitting the screen height and FAQ is moving. This is how website was 10 days ago :smile: 
Please make seperate PR's from now. I could have merged Code of conduct atleast

File: main/index.html [hunk 1]
```diff
@@ -3,12 +3,13 @@
 <head>
     <meta charset="utf-8">
     <meta http-equiv="x-ua-compatible" content="ie=edge">
-    <title>HINT v3.0</title>
+    <title>Hack In The North 3.0 - India's Biggest Student Hackathon</title>
```
Comment (gauthamzz) [inline main/index.html]: @yasharmaster bro this looks so much better than hint

File: main/index.html [hunk 2]
```diff
@@ -45,16 +47,19 @@
                         <div class=" navbar-right">
                             <ul class="nav navbar-nav">
                                 <li>
-                                    <a id="nav_about" href="#about" class="nav-link">ABOUT</a>
+                                    <a id="nav_about" href="#about" class="nav-link"><strong>ABOUT</strong></a>
```
Comment (gauthamzz) [inline main/index.html]: idk why this is done. It was better not bold

File: main/index.html [hunk 3]
```diff
@@ -79,7 +84,7 @@ <h3 class="landing-subtitle">March XX-XX, 2018</h3>
                     <h3 class="landing-subtitle">Indian Institute of Information Technology, Allahabad</h3>
                 </div>
                 <div class="row col-md-12 text-center wow fadeIn" data-wow-delay="1s">
-                    <h4>Applications are now open.</h4>
+                    <h4>Applications are open now.</h4>
```
Comment (gauthamzz) [inline main/index.html]: Thanks

File: main/index.html [hunk 4]
```diff
@@ -144,7 +151,8 @@ <h3 class="timeline__title">Repellendus nostrum alias odit.</h2>
                         </div>
                     </div>
                 </section>
-                <section id="faq" class="fullpage">
```
Comment (gauthamzz) [inline main/index.html]: @Anupam-dagar isnt this the class you added after my request ?

File: main/index.html [hunk 5]
```diff
@@ -246,10 +257,11 @@ <h1 class="contact-title">CONTACT</h1>
                                             </div>
                                         </div>
                                     </section>
+
                                     <section id="footer">
                                         <div class="contents text-center row">
                                             <div class="col-md-6 col-md-offset-3 footer-text">
-                                                ©2017 Hack In The North<br/>
+                                                <a style="color: white" href="https://hackcodeofconduct.org/530-hack_in_the_north_2018">Code of Conduct </a>| ©2017 Hack In The North<br/>
```
Comment (gauthamzz) [inline main/index.html]: This looks good

File: main/static/css/cards.css [hunk 6]
```diff
@@ -1,16 +1,21 @@
 .box{
   border: 3px solid #fff;
 }
+
```
Comment (gauthamzz) [inline main/static/css/cards.css]: Why are these empty lines being added :confused:

File: main/index.html (hunk 4 above)
Comment (Anupam-dagar) [reply main/index.html]: yeah, it makes the faq section a fullpage one.

File: main/index.html (hunk 2 above)
Comment (OrionStar25) [reply main/index.html]: Ill change that back.

File: main/index.html (hunk 4 above)
Comment (OrionStar25) [reply main/index.html]: It was fitting my screen height so i didnt notice that. Ill do something else for this then.
fullpage class being used is same as the one being used in fullpage.js?

File: main/static/css/cards.css (hunk 6 above)
Comment (OrionStar25) [reply main/static/css/cards.css]: It made the code more readable for me. :P

Comment (gauthamzz) [reply main/static/css/cards.css]: Please don't do these .

File: main/index.html (hunk 4 above)
Comment (gauthamzz) [reply main/index.html]: Change the screen size and see . Always . Being responsive is our number one priority

Comment (Anupam-dagar) [reply main/index.html]: No fullpage js is not linked in the website, fullpage class has this definition
.fullpage{
height: 90vh;
}

Comment (gauthamzz) [reply main/index.html]: Please revert this back. This was not the meaning of the issue

**Pull request closed without merging**